.github/workflows/node-zxc-build-release-artifact.yaml

```diff
@@ -532,6 +537,7 @@ jobs:
     needs:
       - validate
       - build-artifact
+    if: ${{ !github.event.workflow_call.repository.fork }}
```

Comment:
Not needed because the `build-artifact` job is skipped and this workflow requires/needs the `build-artifact` job. Either add as a job level `if` statement for belt and suspenders or remove entirely. Former recommended.